lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -50,43 +50,78 @@ local function MakePropNoEffect(...)
 	return ret
 end
 
-function PropCore.CreateProp(self,model,pos,angles,freeze)
-	if(!util.IsValidModel(model) || !util.IsValidProp(model) || not PropCore.ValidSpawn() )then
-		return nil
+function PropCore.CreateProp(self,model,pos,angles,freeze,isVehicle)
+
+	if not PropCore.ValidSpawn() then return nil end
+	
+	if isVehicle then
+		if self.player:CheckLimit( "vehicles" ) == false then return end
+		if model == "" then model = "models/nova/airboat_seat.mdl" end
+		if model ~= "models/vehicles/prisoner_pod_inner.mdl" then angles:RotateAroundAxis( angles:Up(), -90 ) end
```

Comment:
I don't know if RotateAroundAxis is a good idea, since people might have addons that add chairs which are not rotated weirdly. What do you think @Nebual